### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by ivandmitrienko**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      activeIndex: null,
+      activeChevron: '',
+    };
+  }
+
+
+  handleclick = (itemsId) => {  
+    return itemsId===this.state.activeIndex ? 
+    this.setState({ activeIndex: null, activeChevron: ''}): 
+    this.setState({ activeIndex: itemsId, activeChevron: 'active', });
```
Comment (DmitryMoodys) [inline src/App.jsx]: Could be optimised :) 

```
const newState = itemsId===this.state.activeIndex ? { activeIndex: null, activeChevron: ''} : { activeIndex: itemsId, activeChevron: 'active', };

this.setState(newState);
```

**Comment by DmitryMoodys** [comment]
Could you try to use this Code Formatting Shortcuts: `Ctrl + Shift + I` for every file you are dealing with?
It will make the code more readable and  "beautiful"

https://mkyong.com/vscode/how-to-format-source-code-in-visual-studio-code-vscode/

File: src/Block.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+.block__title{
+    height: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 15px; 
+    font-size: 14px;
+  
+
```
Comment (DmitryMoodys) [inline src/Block.css]: empty lines :)

**Comment by DmitryMoodys** [comment]
The rest looks good!
Great job!

File: src/Block.css (hunk 2 above)
Comment (ivandmitrienko) [reply src/Block.css]: Thanks, Bro)

**Pull request merged**